docs/Customization/RoCC-Accelerators.rst

```diff
@@ -46,6 +50,47 @@ the ``busy`` signal, which indicates when the accelerator is still handling an i
 and the ``interrupt`` signal, which can be used to interrupt the CPU.
 
 Look at the examples in ``generators/rocket-chip/src/main/scala/tile/LazyRoCC.scala`` for detailed information on the different IOs.
+There is also more information about each of the signals in `the RoCC Documentation written by UCSD <https://docs.google.com/document/d/1CH2ep4YcL_ojsa3BVHEW-uwcKh1FlFTjH_kg5v8bxVw/edit>`_, although it is updated out of tree and may be out of date.
+
+
+Accessing Memory via L1 Cache
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A RoCC accelerator can access memory through the L1 Cache of the core it is attached to.
+This is typically a good idea, as a RoCC instruction that passes in an address likely has accessed it recently.
+
+In your ``LazyRoCCModuleImp``, the signal ``io.mem`` is a ``HellaCacheIO``, which is defined in ``generators/rocket-chip/src/main/scala/rocket/HellaCache.scala``.
+If your core is a RocketCore, the other end of the input is connected to a ``SimpleHellaCacheIF``, which has a replay buffer in case the L1 cache is overloaded and doesn't ACK your request.
```

Comment:
Can you omit mentioning the SimpleHellaCacheIF, that is an implementation detail. You should emphasize the point that only the `req` and `resp` signals matter for the accelerator implementation, the rest can be ignored.